LabExT/View/Movement/CoordinatePairingsWindow.py

```diff
@@ -233,6 +236,24 @@ def _on_device_selection_clear(self) -> None:
         self._device = None
         self.__reload__()
 
+    def _ask_user_move_to_device(self):
+        """
+        Asks the user whether to move to the device after selecting the device. This is possible after a single pairing.
+        """
+        if self._device is None or not self.mover.can_move_absolutely:
+            return
+
+        if not messagebox.askyesno(
+                title="Move to device?",
+                message="Your stages can move absolutely in the chip coordinate system. Do you want to move to the device?",
```

Comment:
Rephrase: "Do you want to move the stages close to the selected device using the available coarse calibration?"